document/AnnexC.mdk

```diff
@@ -50,6 +50,8 @@ allows viewer implementers to assume that all files from the same version of the
 
 This general guidance is embodied in various sections of the specification. For example:
 
+* A converter should not attempt to synthesize a `ruleId` for a result if the tool does not provide one.
```

Comment:
I found another example in the of the guidance "don't synthesize stuff", and replicated it here in the Annex about converters.